neurovault/apps/statmaps/views.py

```diff
@@ -648,11 +648,11 @@ def delete_image(request, pk):
 
 def view_images_by_tag(request, tag):
     if request.user.is_authenticated():
-        images = Image.objects.filter(tags__name__in=[tag]).filter(
+        images = Image.objects.filter(tags__name__in=[tag.lower()]).filter(
                                         Q(collection__private=False) |
                                         Q(collection__owner=request.user))
     else:
-        images = Image.objects.filter(tags__name__in=[tag]).filter(
+        images = Image.objects.filter(tags__name__in=[tag.lower()]).filter(
```

Comment:
If an image is tagged with "Brain" or "BRAIN" it will not  be captured by this query (for tag=="brain").